parser/src/parser/tests/testdata/with-2.cst

```diff
@@ -0,0 +1,104 @@
+SOURCE_FILE@0..162
+  RULE_DECL@0..162
+    RULE_KW@0..4 "rule"
+    WHITESPACE@4..5 " "
+    IDENT@5..9 "test"
+    WHITESPACE@9..10 " "
+    L_BRACE@10..11 "{"
+    NEWLINE@11..12 "\n"
+    WHITESPACE@12..13 "\t"
+    CONDITION_BLK@13..160
+      CONDITION_KW@13..22 "condition"
+      COLON@22..23 ":"
+      NEWLINE@23..24 "\n"
+      WHITESPACE@24..26 "\t\t"
+      BOOLEAN_EXPR@26..160
+        BOOLEAN_TERM@26..160
+          WITH_EXPR@26..160
+            WITH_KW@26..30 "with"
+            WHITESPACE@30..31 " "
+            WITH_IDENTIFIERS@31..90
```

Comment:
The `WITH_IDENTIFIERS` rule contains a sequence of comma-separated declarations in the form `identifier = expr`, however I miss some grammar rule that contains each individual declarations. Something like:

``
WITH_IDENTIFIERS :=  WITH_IDENTIFIER [, WITH_IDENTIFIER]*
``

But as mentioned before, I would change change the names of these rules as: 

```
WITH_DECLS :=  WITH_DECL [`,` WITH_DECL]*

WITH_DECL := IDENT `=` EXPR
```

Reply:
Added in: https://github.com/VirusTotal/yara-x/pull/197/commits/3858aafe1452bba33aa2334bc6da55ced10a15f9